File: 300-399/395 Donation Rewards DOREWARD.c
//x --> no. of donations

//x<=3 --> BRONZE
//x>3 && x<=6 --> SILVER
//X>6 --> GOLD
#include <stdio.h>
int main(void)
{
	int t;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    int x;
	    scanf("%d",&x);
	    if(x<=3)
	    {
	        printf("BRONZE\n");
	    }
	    else if(x>3 && x<=6)
	    {
	        printf("SILVER\n");
	    }
	    else if(x>6)
	    {
	        printf("GOLD\n");
	    }
	}
	return 0;
}

File: Code Chef/JavaScript/300-399/395 Donation Rewards DOREWARD.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    const intArray = strArray.map(Number);
    return intArray;
};

const badgeByDonation = (bloodDonations) => {
    (bloodDonations <= 3)
        ? console.log("BRONZE")
        : ((bloodDonations > 3) && (bloodDonations <= 6))
            ? console.log("SILVER")
            : console.log("GOLD");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n").slice(1));

    testcases.forEach((bloodDonations) => {
        badgeByDonation(bloodDonations);
    });
});
